digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -361,6 +364,12 @@ CommandStatus createNextCommandStatus(
             }
             while (Instant.now().getEpochSecond() < timeout);
 
+            if (previousExecutorStatus.get("logging_finished_at") == null) {// case for timeout
+                logger.warn(s("Cannot find logging_finished_at marker in CloudWatch logs.\n"
```

Comment:
IIRC, `warn` level is visible to customer. My concern is some customer might be worried too much when seeing this warning. What do you think?

Reply:
don't have strong opinion. changed to debug level.